### PR TITLE
Leverage colour scheme from theme.json

### PR DESCRIPTION
**Pull request by jaymcp**
Ref: https://github.com/amnestywebsite/humanity-theme/issues/335

We currently define a colour scheme in `theme.json`, but do not make use of the CSS vars generated by it.

This PR replaces our SCSS colour variables with the CSS vars. It also removes some unused colours within theme.json, and names them more clearly.

**Steps to test**:
1. checkout the branch
2. cd into the `./private` directory
3. run `yarn build`
4. check the following:
   - sections have grey backgrounds
   - the tweet action block alternate style uses the Twitter brand colour for its button bg
   - button colours are correct (default is primary colour; dark is... dark)
   - news index filters and list container have grey backgrounds
   - hero text background colours are correct
   - etc.

### CODE REVIEW

File: wp-content/themes/humanity-theme/theme.json [hunk 1]
```diff
@@ -20,127 +20,82 @@
...
           "color": "#767676",
-          "name": "Grey Base"
+          "name": "Default Grey"
         },
         {
-          "slug": "amnesty-grey-mid-dark",
+          "slug": "grey-dark",
           "color": "#525252",
-          "name": "Grey Mid Dark"
+          "name": "Dark Grey"
```
Comment (mikachan) [inline wp-content/themes/humanity-theme/theme.json:65]: Dark Grey and Dark are the same colour - Could this be reduced to one colour value?

**Review by mikachan — COMMENTED** [review]
This is mainly a smoke test as I'm not too familiar with the theme, but hopefully it's an additional confidence check!

- I can't see any references to the old colour names anywhere in the theme ✅ 
- I think the new colour naming looks good in comparison to the old names ✅ 
- The CSS builds successfully ✅ 

I see that these colours have been removed from both the SCSS and theme.json file, but they're also not used anywhere:

- $color-grey-x-light: #eee;
- $color-grey-sm-light: #e7e7e7;
- $color-grey-md-light: #b5b5b5;
- $color-grey-mid-light: #717272;
- $color-figure-bgd: #f5f5f5;
- All the blue colours

I can't see the Amnesty logo yellow colour (#fdff28), is that because it's not used anywhere in the CSS? I'm wondering if it would still be helpful to have in the palette, but perhaps not if it's not used anywhere.

**Comment by jaymcp** [comment]
> This is mainly a smoke test as I'm not too familiar with the theme, but hopefully it's an additional confidence check!

Thanks for taking a look!

> I can't see the Amnesty logo yellow colour (#fdff28), is that because it's not used anywhere in the CSS? I'm wondering if it would still be helpful to have in the palette, but perhaps not if it's not used anywhere.

We define Amnesty branding colours externally to this theme, as we intend for the theme to be purely generically branded. I can provide you access to the repo containing these definitions if it would be useful to you, just let me know.

File: wp-content/themes/humanity-theme/theme.json (hunk 1 above)
Comment (jaymcp) [reply wp-content/themes/humanity-theme/theme.json:65]: Good spot! I will take a further look at the dark colours.

**Comment by mikachan** [comment]
> We define Amnesty branding colours externally to this theme

Right, I forgot! That makes sense 👍

File: wp-content/themes/humanity-theme/theme.json (hunk 1 above)
Comment (jaymcp) [reply wp-content/themes/humanity-theme/theme.json:65]: I actually couldn't see any of the "dark" colours in use anywhere, so I've removed all three in 7859106

**Review by Gentyspun — APPROVED** [review]
LGTM 👍

**Pull request merged**